bgpd/bgp_ovsdb_if.c

```diff
@@ -2122,10 +2122,6 @@ void bgp_daemon_ovsdb_neighbor_update (struct peer *peer,
         &peer->local_as, 1);
      */
 
-    VLOG_DBG("updating weight to %d\n", peer->weight);
-    ovsrec_bgp_neighbor_set_weight(ovs_bgp_neighbor_ptr,
```

Comment:
Which part exactly? This one was removed because BGP neighbor weight is a configuration parameter and it is not distributed to BGP neighbor. Basically it has only local meaning and only when a best route must be selected from BGP route table. There is no reason for BGPD to update this parameter upstream in OVSDB, it should only be changed by user and propagated to BGPD so it could assign weights to routes from different neighbors.